lib/miq_automation_engine/engine/miq_ae_method.rb

```diff
@@ -276,41 +283,29 @@ def self.invoke_inline_ruby(aem, obj, inputs)
     def self.run_method(cmd)
       require 'open4'
       rc = nil
-      final_stderr = []
-      threads = []
       method_pid = nil
       begin
         status = Open4.popen4(*cmd) do |pid, stdin, stdout, stderr|
           method_pid = pid
           yield stdin if block_given?
           stdin.close
-          threads << Thread.new do
-            stdout.each_line { |msg| $miq_ae_logger.info "Method STDOUT: #{msg.strip}" }
-          end
-          threads << Thread.new do
-            stderr.each_line do |msg|
-              msg = msg.chomp
-              final_stderr << msg
-              $miq_ae_logger.error "Method STDERR: #{msg}"
-            end
-          end
-          threads.each(&:join)
+          stdout.each_line { |msg| $miq_ae_logger.info "Method STDOUT: #{msg.strip}" }
```

Comment:
@bdunne 
This issue might also happen on the container methods if we can't launch the method due to some system errors, missing dependencies etc. Anything it does before rerouting the STDERR,STDOUT could get us blocked.